content/applications/productivity/iot/config/https_certificate_iot.rst

```diff
@@ -0,0 +1,217 @@
+
+Solution:
+    Check if `<https://www.odoo.com>`__ is not down using a browser, it might be down
+    due to maintenance.
+
+     - If it is in maintenance, you will have to wait for it to be back
+     - If it is not, we invite you to open a
+       `support ticket <https://www.odoo.com/help>`__ on the matter. Don't
+       forget to mention on it the 3 digits status code next to the code error
+
+`ERR_IOT_HTTPS_LOAD_REQUEST_NO_RESULT`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Reason:
+    The IoT-box was able to reached `<https://www.odoo.com>`__ but it
+    refused to deliver the HTTPS certificate.
+
+Solution:
+    Please make sure that you are eligible: `IoT-box eligibility`_
+
+
+How do I know if the HTTPS certificate is correct ?
+---------------------------------------------------
+
+If the certificate is applied successfully, you will see on your odoo database
+a new HTTPS URL for the IoT-box ending with `.odoo-iot.com`.
+
+.. image:: https_certificate_iot/odoo_new_domain.png
+   :align: center
+
+If you browse to this link you should see in your browser that a secured
+HTTPS connection was established.
+
+.. figure:: https_certificate_iot/secured_connection.png
+   :align: center
+
+   Padlock in Chrome on Windows 10 attesting the fact that the connection is secured in HTTPS
+
+The IoT-box homepage will also now display an `OK` status next to `HTTPS certificate`.
+The :guilabel:`details` button next to it will give information regarding the certificate.
+
+.. image:: https_certificate_iot/status_ok.png
+   :align: center
+
+
+DNS issue
+---------
+
+If you can access your IoT-box from it's IP address but not its
+`.odoo-iot.com` one. You are likely running into a DNS issue.
+On some browsers, it will give an error code mentioning `DNS`
+(like `DNS_PROBE_FINISHED_NXDOMAIN`).
+Such issues look as follows:
+
+.. figure:: https_certificate_iot/dns/dns_chrome.png
+   :align: center
+
+   DNS issue on Chrome browser on Windows 10
+
+.. figure:: https_certificate_iot/dns/dns_firefox.png
+   :align: center
+
+   DNS issue on Firefox browser on Windows 10
+
+.. figure:: https_certificate_iot/dns/dns_edge.png
+   :align: center
+
+   DNS issue on Edge browser on Windows 10
+
+DNS issue Solution
+~~~~~~~~~~~~~~~~~~
+
+ #. If your router allow you to change the DNS manually on it.
+    Set it to use `Google DNS <https://developers.google.com/speed/public-dns>`__.
+ #. If not, then you will have to change the DNS settings on each
+    devices for `Google DNS <https://developers.google.com/speed/public-dns>`__.
+    It will have to be done on **every** device that plan to interact with
+    the IoT-box (computer, tablet and phone).
+    We invite you to check online per device how to modify its DNS setting.
+
+    .. note::
+       Other IoT devices such as payment terminals will likely don't need
+       to change their DNS settings as they are already configured
+       with custom DNS.
```

Comment:
```suggestion
    .. note::
       Other :abbr:`IoT (Internet of Things)` devices such as payment terminals likely won't need
       their :abbr:`DNS (Domain Name System)` settings changed as they are already configured with custom :abbr:`DNS (Domain Name System)`.
```

Do the payment terminals come from the factory with custom DNS or is this achieved during setup?

Reply:
it depends, but generally it is factory (I think)